src/scribe.js

```diff
@@ -179,10 +181,19 @@ define([
     return this.el.textContent;
   };
 
+  /**
+   * Replace the current undo manager. Can be used by undo manager plugins or directly by the user.
+   * @param {Object} undoManager The undo manager instance to be used by the current editor.
+   */
+  Scribe.prototype.setUndoManager = function (undoManager) {
+	  this.undoManager = undoManager;
```

Comment:
We use two spaces for alignment in scribe. You should be able use to editorconfig to sort this out for you...